### PR TITLE
Contact page

### PR DESCRIPTION
**Pull request by Mussuranae**
Contact page for dashboard
Implementing google map for the view and try to develop function to send email from user to blockframes admin with sendgrid

### CODE REVIEW

File: apps/backend-functions/src/users.ts [hunk 1]
```diff
@@ -180,3 +180,13 @@ export const sendDemoRequest = async (
 
   return data;
 }
+
+export const sendUserMail = async (data: any, context?: CallableContext): Promise<any> => {
+  const { userName, userMail, subject, message } = data;
+
+  if (!subject || ! message || userName || userMail) {
```
Comment (pldespaigne) [inline apps/backend-functions/src/users.ts]: you forgot `!` before `userName` & `userMail`

File: apps/catalog/catalog/src/app/dashboard/pages/contact-page/contact-page.component.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
-import { ChangeDetectionStrategy, Component } from "@angular/core";
+import { ChangeDetectionStrategy, Component, OnInit } from "@angular/core";
+import { FormGroup, FormControl, Validators } from "@angular/forms";
+import { AngularFireFunctions } from "@angular/fire/functions/functions";
```
Comment (pldespaigne) [inline apps/catalog/catalog/src/app/dashboard/pages/contact-page/contact-page.component.ts]: try with `import { AngularFireFunctions } from '@angular/fire/functions';`

File: apps/backend-functions/src/users.ts (hunk 1 above)
Comment (Mussuranae) [reply apps/backend-functions/src/users.ts]: Good catch, I'm really not attentive -_-

File: apps/backend-functions/src/assets/mail-templates.ts [hunk 3]
```diff
@@ -182,7 +182,7 @@ export function sendDemoRequestMail(information: RequestDemoInformations) {
 
 export function sendContactEmail(userName: string, userMail: string, subject: string, message: string): EmailRequest {
   return {
-    to: adminEmail,
+    to: 'cmandonnet@cascade8.com',
```
Comment (pldespaigne) [inline apps/backend-functions/src/assets/mail-templates.ts]: don't forget to put back `adminEmail` after the tests

Comment (Mussuranae) [reply apps/backend-functions/src/assets/mail-templates.ts]: Look last commit x) I think about it just after push the 'last' version !

**Pull request merged**